source/NVDAObjects/window/__init__.py

```diff
@@ -399,19 +399,20 @@ def event_valueChange(self):
 		# Don't report value changes for editable text fields.
 		pass
 
-class DisplayModelLiveText(LiveText, Window):
-	TextInfo = displayModel.EditableTextDisplayModelTextInfo
-
+class DisplayModelLiveTextWithoutTextInfo(LiveText, Window):

```

Comment:
I think this name is confusing. How about:

```suggestion
class LiveTextWithDisplayModelMonitoring(LiveText, Window):
```

I  once started with splitting LiveText into a more abstract TextMonitor class, but decided to close that pr for now. See #8414 

I would love it if you could consider grabbing this TextMonitor class from #8414. Than you could have the following structure:

1. Have an abstract TextMonitor
2. Have a DisplayModelTextMonitor. In the pr, that is now in the window module. It makes sense to move it to behaviors and remove the TextInfo override.
3. Have LiveText inherrit from TextMonitor
4. Have DisplayModelLiveText inherrit from DisplayModelTextMonitor and LiveText, and override its textInfo

In the Miranda appModule, it then suffices to add LiveText and DisplayModelTextMonitor as overlay classes to the convrsation view. So, you do not have to add an extra, slightly confusing class to the window module.
Note that DisplaymodelTextMonitor's doc string should clearly read that it uses the display model for text monitoring, while not using the display model for the TextInfo. In other words, if the display model notifies of changes, the assumption is made that the unlderying TextInfo implementation for the control will expose the new text.

I really like your proposal of using the Display Model this way. I think it can open some extra possibilities.